### PR TITLE
Add a custom value to set default flyspell correct direction.

### PR DESCRIPTION
**Pull request by smcallis**
Hi I tweaked the script so that I can set the default correction direction.  I usually ctrl-arrow up to get my cursor somewhere north of things and then run stuff so being able to correct forwards is a biggie for me.  Thought you might like to have the change.

### CODE REVIEW

**Comment by d12frosted** [comment]
Hey @smcallis 

Thanks for your PR. It's nice to have an ability for users to change default direction of correction. I remember there was a conversation about this some time ago in [this thread][1]. But I never found time to implement it.

Since a lot of time passed since then, maybe it worth just merging your PR and call it a day 😸 But if you wish to implement the proposed solution, I would be glad 😸 

WDYT?

[1]: https://github.com/d12frosted/flyspell-correct/issues/81#issuecomment-685804682

**Comment by smcallis** [comment]
Happy to bang out a quick change however you like, could you summarize the final thinking in that thread on how people would like this to work?
